telosnetwork/telos-wallet
# Replacing the old ConnectWalletOptions for the current EVMLoginButtons

# Fixes #515

## Description
read [here](https://github.com/telosnetwork/telos-wallet/issues/515)

## Test scenarios
- go to [this NFT page](https://deploy-preview-516--wallet-staging.netlify.app/evm/collectible-details?contract=0xFebd6214aeE0391590d46EfF558e6f43833D1Dc5&id=7&trace=true) (make sure you are not logged)
- try to log in using the side menu button
  - the component popping up should not be the old ConnectWalletOptions but rather the EVMLoginButtons

## Screeshot
![image](https://github.com/telosnetwork/telos-wallet/assets/4420760/6a8ce8ed-4fca-4253-905e-11d9797e7fe5)


## Checklist:
